dcrjson/chainsvrresults.go

```diff
@@ -265,11 +393,179 @@ type GetTxOutResult struct {
+	Mean   float64 `json:"mean"`
+	Median float64 `json:"median"`
+	StdDev float64 `json:"stddev"`
+}
+
+// FeeInfoRange is ticket fee information about a range.
+type FeeInfoRange struct {
+	Number uint32  `json:"number"`
+	Min    float64 `json:"min"`
+	Max    float64 `json:"max"`
+	Mean   float64 `json:"mean"`
+	Median float64 `json:"median"`
+	StdDev float64 `json:"stddev"`
+}
+
+// FeeInfoWindow is ticket fee information about an adjustment window.
+type FeeInfoWindow struct {
+	StartHeight uint32  `json:"startheight"`
+	EndHeight   uint32  `json:"endheight"`
+	Number      uint32  `json:"number"`
+	Min         float64 `json:"min"`
+	Max         float64 `json:"max"`
+	Mean        float64 `json:"mean"`
+	Median      float64 `json:"median"`
+	StdDev      float64 `json:"stddev"`
+}
+
+// TicketFeeInfoResult models the data returned from the ticketfeeinfo command.
+// command.
+type TicketFeeInfoResult struct {
+	FeeInfoMempool FeeInfoMempool  `json:"feeinfomempool"`
+	FeeInfoBlocks  []FeeInfoBlock  `json:"feeinfoblocks"`
+	FeeInfoWindows []FeeInfoWindow `json:"feeinfowindows"`
+}
+
+// SearchRawTransactionsResult models the data from the searchrawtransaction
+// command.
+type SearchRawTransactionsResult struct {
+	Hex           string       `json:"hex,omitempty"`
+	Txid          string       `json:"txid"`
+	Version       int32        `json:"version"`
+	LockTime      uint32       `json:"locktime"`
+	Vin           []VinPrevOut `json:"vin"`
+	Vout          []Vout       `json:"vout"`
+	BlockHash     string       `json:"blockhash,omitempty"`
+	Confirmations uint64       `json:"confirmations,omitempty"`
+	Time          int64        `json:"time,omitempty"`
+	Blocktime     int64        `json:"blocktime,omitempty"`
+}
+
+// TxFeeInfoResult models the data returned from the ticketfeeinfo command.
+// command.
+type TxFeeInfoResult struct {
+	FeeInfoMempool FeeInfoMempool `json:"feeinfomempool"`
+	FeeInfoBlocks  []FeeInfoBlock `json:"feeinfoblocks"`
+	FeeInfoRange   FeeInfoRange   `json:"feeinforange"`
+}
+
+// TicketsForAddressResult models the data returned from the ticketforaddress
+// command.
+type TicketsForAddressResult struct {
+	Tickets []string `json:"tickets"`
+}
+
+// ValidateAddressChainResult models the data returned by the chain server
+// validateaddress command.
+type ValidateAddressChainResult struct {
+	IsValid bool   `json:"isvalid"`
+	Address string `json:"address,omitempty"`
+}
+
+// VersionResult models objects included in the version response.  In the actual
+// result, these objects are keyed by the program or API name.
+type VersionResult struct {
+	VersionString string `json:"versionstring"`
+	Major         uint32 `json:"major"`
+	Minor         uint32 `json:"minor"`
+	Patch         uint32 `json:"patch"`
+	Prerelease    string `json:"prerelease"`
+	BuildMetadata string `json:"buildmetadata"`
+}
+
+// CreateMultiSigResult models the data returned from the createmultisig
+// command.
+type CreateMultiSigResult struct {
```

Comment:
There are a lot of results that aren't alphabetized here.  The only time they shouldn't be is if  they are subresults of a field of a specific command in which case that subresult should come before the command result that uses it.